scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/model/impl/ProjectImpl.java

```diff
@@ -0,0 +1,81 @@
+package edu.asu.diging.scriptoocloud.core.model.impl;
+
+import java.time.ZonedDateTime;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import edu.asu.diging.scriptoocloud.core.model.Project;
+
+@Entity
+public class ProjectImpl implements Project {
```

Comment:
Okay. Should I add it as a string to the project itself? String projectCreator and function setProjectCreator? In turn, shall I also have a string and set function for someone who deletes a project?

Reply:
Since you have a multi-user system, how did you plan to keep track of all the projects of a user? Take a look at some of the other classes, that should answer your questions I think.